src/quick-lint-js/fe/global-variables.cpp

```diff
@@ -232,6 +232,8 @@ constexpr const char8 global_variables_node_js_es[] =
     u8"setTimeout\0"
     u8"unescape\0";
 
+constexpr const char8 global_variables_quickjs[] = u8"scriptArgs\0";
```

Comment:
I think this should include `console` and `print` too. Otherwise, if someone configures quick-lint-js for ecmascript+quickjs, `console.log` would warn.